src/components/Button/index.stories.tsx

```diff
@@ -214,7 +220,7 @@ Loading.parameters = {
   },
 };
 
-export const Breakout = (args) => (
+export const Breakout = (args: ButtonProps): JSX.Element => (
```

Comment:
Hmm... I see here that this might go against my comment above. I wonder if Storybook has a type they offer for this.

Reply:
My perception is that the story itself doesn't have much to do with storybook itself... it's just a component for the most part, but yeah let me know if you find anything.